package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "requirejs-plugins-current",
+  "version": "1.0.3",
+  "description": "Small set of plugins for RequireJS (http://requirejs.org). Some plugins may also work on other AMD loaders (never tested it). For more plugins check RequireJS Wiki (https://github.com/jrburke/requirejs/wiki/Plugins). Note: this is a fork of the original requirejs-plugins repository, which only adds the package.json to make it installable via npm.",
+  "keywords": "require, requirejs, plugins, node",
```

Comment:
Keywords should be an array of strings (see https://docs.npmjs.com/files/package.json#keywords).